_episodes/06-style-guide.md

```diff
@@ -3,13 +3,24 @@ title: "Style Guide"
 teaching: 10
 exercises: 0
 questions:
+- "What style should be used for Carpentries lessons?"
 - "How are keyboard key combinations written?"
+- "How are short spans of code written?"
 objectives:
 - "Explain keyboard key combination."
+- "Locate documentation for general matters of style in The Carpentries."
+- "Identify uses of the span of code markdown feature."
 keypoints:
-- "Keyboard keys need to use `<kbd>` HTML tag."
+- "The Carpentries Handbook contains a style guid."
+- "Keyboard actions are described using the `<kbd>` HTML tag."
+- "The backtick (_\`_) is used as a span of code indicator to enclode code, filenames, and 
+  text to type that appears within in narrative text."
 ---
 
+This episode contains lesson-specifc style guidance and related implementation details related to The Carpentries lesson template.  For more general matters of style, see the 
+[Style Guide](https://docs.carpentries.org/topic_folders/communications/style-guide.html) 
```

Comment:
We could just say `[The Carpentries Style Guide](https://docs.carpentries.org/topic_folders/communications/style-guide.html)`, omitting `in The Carpentries Handbook`.